app.py

```diff
@@ -1,33 +1,85 @@
-from flask import Flask
+from flask import Flask, g, json, jsonify, render_template, redirect, request, url_for
 from flask_cors import CORS
 from werkzeug import exceptions
 import sqlite3
+from controllers import urls
+
 
 app = Flask(__name__)
 CORS(app)
 
 DATABASE = 'database/database.db'
+URL = 'shoter.io'
```

Comment:
Seems like you have used Short.io - interesting choice and nice to see branching out of new technologies. However I couldn't seem to get the short.io link to redirect to original link

Reply:
I haven't fully investigated short.io but perhaps are you missing some initial set up? This is the short.io API documentation I found https://developers.short.io/docs/adding-a-domain - it seems to suggest that you need an API key and have to install/import some things?